sample_info/modules/stability.py

```diff
@@ -11,6 +11,13 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
+"""
+TODO: write description and notes.
```

Comment:
Take care of the `TODO`.

Reply:
Added a single-sentence description of `stability.py`.